--- origsrc/enchant-1.6.0/configure.in	2010-04-01 15:56:07.000000000 -0500
+++ src/enchant-1.6.0/configure.in	2015-08-28 12:24:43.098800100 -0500
@@ -287,15 +287,6 @@ if test "x$have_cxx" = "xno"; then
 fi
 AM_CONDITIONAL(WITH_MYSPELL, test "x$build_myspell" = "xyes")
 
-myspell_dir=${datadir}/myspell/dicts
-AC_ARG_WITH(myspell-dir, AS_HELP_STRING([--with-myspell-dir=PATH],[path to installed myspell dicts]))
-
-if test "x$with_myspell_dir" != "x" ; then
-   myspell_dir=$with_myspell_dir
-fi
-
-MYSPELL_CFLAGS="$MYSPELL_CFLAGS -DENCHANT_MYSPELL_DICT_DIR='\"$myspell_dir\"'"
-
 AC_ARG_WITH(system-myspell, AS_HELP_STRING([--with-system-myspell=yes/no],[use the system myspell/hunspell @<:@default=auto@:>@]))
 if test "x$with_system_myspell" != "xno"; then
 	PKG_CHECK_MODULES(MYSPELL, [hunspell], have_system_myspell=true, have_system_myspell=false)
@@ -306,6 +297,16 @@ else
 fi
 AM_CONDITIONAL(WITH_SYSTEM_MYSPELL, test "x$have_system_myspell" = "xtrue")
 
+myspell_dir=${datadir}/myspell/dicts
+AC_ARG_WITH(myspell-dir, AS_HELP_STRING([--with-myspell-dir=PATH],[path to installed myspell dicts]))
+
+if test "x$with_myspell_dir" != "x" ; then
+   myspell_dir=$with_myspell_dir
+fi
+
+MYSPELL_CFLAGS="$MYSPELL_CFLAGS -DENCHANT_MYSPELL_DICT_DIR='\"$myspell_dir\"'"
+AC_SUBST(MYSPELL_CFLAGS)
+
 check_aspell=yes
 build_aspell=no
 
