--- origsrc/enchant-1.4.2/configure.in	2008-05-05 15:26:54.000000000 -0500
+++ src/enchant-1.4.2/configure.in	2008-05-07 00:26:21.062500000 -0500
@@ -293,8 +293,8 @@
    myspell_dir=$with_myspell_dir
 fi
 
-MYSPELL_CFLAGS="$MYSPELL_CFLAGS -DENCHANT_MYSPELL_DICT_DIR='\"$myspell_dir\"'"
 PKG_CHECK_MODULES(MYSPELL, [hunspell], have_system_myspell=true, have_system_myspell=false)
+MYSPELL_CFLAGS="$MYSPELL_CFLAGS -DENCHANT_MYSPELL_DICT_DIR='\"$myspell_dir\"'"
 AC_SUBST(MYSPELL_CFLAGS)
 AC_SUBST(MYSPELL_LIBS)
 AM_CONDITIONAL(WITH_SYSTEM_MYSPELL, test "x$have_system_myspell" = "xtrue")
--- origsrc/enchant-1.4.2/src/zemberek/Makefile.am	2008-05-05 15:19:19.000000000 -0500
+++ src/enchant-1.4.2/src/zemberek/Makefile.am	2008-05-07 00:26:21.078125000 -0500
@@ -11,7 +11,7 @@
 
 libenchant_zemberek_lalibdir=$(libdir)/enchant
 libenchant_zemberek_la_LIBADD= $(ZEMBEREK_LIBS) $(top_builddir)/src/libenchant.la
-libenchant_zemberek_la_LDFLAGS = -avoid-version -no-undefined
+libenchant_zemberek_la_LDFLAGS = -module -avoid-version -no-undefined
 
 libenchant_zemberek_la_SOURCES =\
 	zemberek.cpp		\
